independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/startup/RuntimeDeploymentManager.java

```diff
@@ -127,6 +139,27 @@ public BeanFactory.BeanInstance<?> apply(Class<?> aClass) {
             }
             Map<String, RequestMapper<RuntimeResource>> mappersByMethod = new RuntimeMappingDeployment(templates)
                     .buildClassMapper();
+            for (Map.Entry<String, RequestMapper<RuntimeResource>> entry : mappersByMethod.entrySet()) {
```

Comment:
I'd like to avoid iterating over entries of a map. There are some examples in this class that use Map.forEach instead

Reply:
I remember you being against using lambda expressions in runtime code (at some point in the past anyway), so I think I avoided forEach basically for that reason ;)

Reply:
In general yes, but the fact is that iterating of maps is even worse :(. `Map.forEach` allows for optimal map access and when used with method references should lead to minimal metaspace overhead

Reply:
Ok, changed that to `forEach`